src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/EventLoggingEntity.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.personrecord.jpa.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.LocalDateTime
+import java.util.*
```

Comment:
please can we not have this * import? IntelliJ is configured by default to allow this one for some reason